src/Concerns/ManagesSessionPool.php

```diff
@@ -42,11 +42,7 @@ trait ManagesSessionPool
     public function clearSessionPool(): void
     {
         $sessionPool = $this->getSpannerDatabase()->sessionPool();
-        $operation = $sessionPool?->clear();
-
-        if ($operation instanceof DeleteOperation) {
```

Comment:
Cloud you remove use `Colopl\Spanner\Session\DeleteOperation` ?

Reply:
done